victron_ble/devices/smart_lithium.py

```diff
@@ -0,0 +1,98 @@
+from typing import Optional
+
+from construct import Array, BitsInteger, BitStruct, ByteSwapped, Padding
+
+from victron_ble.devices.base import Device, DeviceData
+
+from enum import Enum
+
+class BalancerStatus(Enum):
+  UNKNOWN = 0
+  BALANCED = 1
+  BALANCING = 2 //?
```

Comment:
Seems likely. Note that `//` isn't a comment in Python.

Reply:
fixed, thanks

Reply:
FYI - confirmed that 2 = "Balancing", one of my batteries is balancing as we speak ;)

Reply:
I had a chance to check that with the app too...